sample-lnd.conf

```diff
@@ -50,6 +50,19 @@
 ; or want to expose the node at a domain.
 ; externalhosts=my-node-domain.com
 
+; Sets the directory to store Let's Encrypt certificates within
+; letsencryptdir=
```

Comment:
nit: I think we started to provide valid/sane example values for all options in this file.

Reply:
added